specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/virtualWan.json

```diff
@@ -2094,6 +2094,77 @@
         }
       }
     },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/vpnGateways/{gatewayName}/vpnConnections/{connectionName}/vpnLinkConnections/{linkConnectionName}/getikesas": {
+      "post": {
+        "tags": [
+          "vpnLinkConnections"
+        ],
+        "operationId": "VpnLinkConnections_GetIkeSas",
+        "description": "Lists IKE Security Associations for vpn site link connection in the specified resource group.",
+        "parameters": [
+          {
+            "$ref": "./network.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "name": "resourceGroupName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the resource group."
+          },
+          {
+            "name": "gatewayName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the gateway."
+          },
+          {
+            "name": "connectionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the vpn connection."
+          },
+          {
+            "name": "linkConnectionName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the vpn link connection."
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "Accepted and the operation GetIkeSas will complete asynchronously."
+          },
+          "200": {
+            "description": "Request successful. The operation getikesas completed on vpn site link connection.",
+            "schema": {
+              "type": "string"
```

Comment:
Is there a reason why there is no model defined for the response?

Reply:
Hi @filizt,
The changes are similar to VNG Connection GetIkeSas. This operation is a POST call and the final state and result of the operation is via location where the IKE SAs are returned in a string format type.